CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Check on tarantool box initialization before init (now it returns a nice error).
+- Posibility to pass port to `memcached.create()` as a number.
```

Comment:
There should be something to read further. An issue number or, if there is no issue, a pull request number. Or both if the pull request is more interesting from a user perspective than the issue.

Reply:
Only in one item.

Reply:
For example the last item has an issue and a pull request. The pull request has more user friendly description, so I would link both. I don't see a reason to link the commit in this circumstances.

Reply:
I don't see the point in specifying the ticket number and PR in Release Notes and not specifying them in other repositories, but I don't mind.
Updated.

Reply:
> There should be something to read further.

It is the point.

Thank you.